spec/author_spec.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../Classes/author'
```

Comment:
- [ ] Kindly make sure to include tests for all of the created methods in all of the created models and not just the `author` and `game` models, as per [this](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things.md#:~:text=Add%20unit%20tests%20for%20all%20implemented%20methods.) requirement.

Reply:
@ShadyShawkat
thanks for your good and valuable feedback, tests are added